Silicon/MeteorlakePkg/Include/CpuInitDataHob.h

```diff
@@ -65,4 +65,11 @@ typedef struct {
   **/
   UINT8                  PerAtomClusterRatio[CPU_MAX_ATOM_CLUSTERS];
 } CPU_INIT_DATA_HOB;
+///
+/// This HOB is used to pass only the required information from PEI for DXE consumption.
```

Comment:
Should remove comments about PEI/DXE since they aren't relevant to SBL. May cause some confusion.